installOrchestDDocker.sh

```diff
@@ -55,37 +91,35 @@ under "new ssh key"'
 }
 
 #  make sure you have an active git user on either GH or BBKT
-show "checking git config user.email:"
-git config user.email
-if [ $? -ne 0 ]; then
-show "Need to config git  user.email"
-show "Show you how?\n[1]Yes\n[2]No"
+show "checking git config user.email..."
+gitemail=$(git config user.email)
+readGithubMailIfNotExist
+
+isGithub=$(ssh -T git@github.com 2>&1)
+if [[ $isGithub != *"Permission denied"* ]]; then
+show "checking git config user.name..."
+gituser=$(git config user.name)
+  if [ $? -ne 0 ]; then
+      installSshKey
+  fi
+fi
+
+show "would you like to work with this git configuration:\nemail=$gitemail , user=$gituser"
+show "[1]Yes\n[2]No"
     read -p "> " INPSEL
 	case $INPSEL in
 	    "1")
-            show "https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#setting-your-email-address-for-every-repository-on-your-computer"
-            exit
+            printf "{\n\t\"server\":\"$GITPATH\",\n\t\"gitEmail\":\"$gitemail\",\n\t\"devBranch\": \"main\",\n\t\"lockedBranches\":[\"dev\",\"master\",\"main\"]\n}\n" > ${userPath}/settings/git.json
 			;;
 	    "2")
-            exit
+	    readGithubUser;
```

Comment:
email first
+github url
"please type the github url you will be working on for this development project:
(if its youre private github, its probably https://github/{user}, if its a company shared github, its probably something like https://github/mycompany)
 ">https://github.com/"
this means setting would have
server
email
user